Source/WebCore/platform/graphics/mac/controls/InnerSpinButtonMac.mm

```diff
@@ -43,13 +44,25 @@
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(InnerSpinButtonMac);
 
-InnerSpinButtonMac::InnerSpinButtonMac(InnerSpinButtonPart& owningPart, ControlFactoryMac& controlFactory)
+InnerSpinButtonMac::InnerSpinButtonMac(InnerSpinButtonPart& owningPart, ControlFactoryMac& controlFactory, NSStepperCell* stepperCell)
     : ControlMac(owningPart, controlFactory)
+    , m_stepperCell(stepperCell)
 {
 }
 
 IntSize InnerSpinButtonMac::cellSize(NSControlSize controlSize, const ControlStyle&) const
 {
+#if HAVE(NSSTEPPERCELL_INCREMENTING)
+    if ([m_stepperCell respondsToSelector:@selector(setIncrementing:)]) {
+        static const IntSize sizes[] = {
```

Comment:
We probably shouldn't specify a large size, given one doesn't exist. I think it will cause problems with `controlSizeForSize` as-is. We may also need to teach that method to ignore controls that don't support a large size.

Reply:
Oh, I see you force `NSControlSizeRegular` below, so maybe okay?